src/Standards/Generic/Tests/PHP/LowerCaseKeywordUnitTest.inc

```diff
@@ -60,5 +60,17 @@ $c = function() {
     FROM fun();
 }
 
+class AsymmetricVisibility {
+    public(set) int $correctPublicSet;
+    protected(set) int $correctProtectedSet;
+    private(set) int $correctPrivateSet;
+
+    PubliC(SeT) int $wrongCasePublic;
+    PROTECTED(set) array $wrongCaseProtected;
+    protected(sEt) int $wrongCaseProtectedSet;
+    Private(set) int $wrongCasePrivate;
+    private(SET) readonly ?string $wrongCasePrivateSet;
+}
+
 __HALT_COMPILER(); // An exception due to phar support.
```

Comment:
Do we need to also provide some test cases where we specify both the get and set visibilities?

Example from https://www.php.net/manual/en/language.oop5.visibility.php#language.oop5.visibility-members-aviz
```php
<?php
class Book
{
    public function __construct(
        public private(set) string $title,
        public protected(set) string $author,
        protected private(set) int $pubYear,
    ) {}
}
```

Reply:
I've done that in most other PRs, but this sniff doesn't look "around", i.e. it doesn't do any token walking, it just looks at the token passed as `$stackPtr`, so in this case, making the tests more diverse would not add any value (as it wouldn't be testing anything which isn't already being tested).